### PR TITLE
Scheduler mapper

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/COM-1160

### CODE REVIEW

File: internal/pkg/state/state.go [hunk 1]
```diff
@@ -106,6 +107,7 @@ func newState(options *Options) *State {
 	}
 	mappers := []interface{}{
 		variables.NewMapper(mapperContext),
+		schedulerMapper.NewMapper(mapperContext),
```
Comment (webrouse) [inline internal/pkg/state/state.go:110]: Pridany `mapper/scheduler`, ... tu importovany ako `schedulerMapper` (nie ako `scheduler`, lebo taky balicek tu uz je pouzity).

File: internal/pkg/mapper/scheduler/load.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
+package scheduler
+
+import (
+	"github.com/iancoleman/orderedmap"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+)
+
+func (m *schedulerMapper) MapAfterRemoteLoad(recipe *model.RemoteLoadRecipe) error {
```
Comment (webrouse) [inline internal/pkg/mapper/scheduler/load.go:9]: Po `remote load` sa zo scheduler konfiguracie zisti `target` -> ktory config sa ma spustat, a jeho `componentId/configurationId`.

File: internal/pkg/mapper/scheduler/load_test.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	. "github.com/keboola/keboola-as-code/internal/pkg/mapper/scheduler"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+func TestSchedulerMapAfterRemoteLoad(t *testing.T) {
```
Comment (webrouse) [inline internal/pkg/mapper/scheduler/load_test.go:14]: Vid test.

File: internal/pkg/mapper/scheduler/save.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import (
+	"fmt"
+
+	"github.com/iancoleman/orderedmap"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+func (m *schedulerMapper) MapBeforeRemoteSave(recipe *model.RemoteSaveRecipe) error {
```
Comment (webrouse) [inline internal/pkg/mapper/scheduler/save.go:12]: Before `remote save`, sa naopak pridat `componentId/configurationId` do konfiguracie.

File: internal/pkg/mapper/scheduler/save_test.go [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+	"github.com/iancoleman/orderedmap"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	. "github.com/keboola/keboola-as-code/internal/pkg/mapper/scheduler"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+func TestSchedulerMapBeforeRemoteSave(t *testing.T) {
```
Comment (webrouse) [inline internal/pkg/mapper/scheduler/save_test.go:15]: Vid test.

**Review by webrouse — COMMENTED** [review]
Implementovany `mapper.scheduler` + iba unit testy.
Samostatne este nedava zmysel

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**